cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -468,7 +468,7 @@ private void onErrorTask(ConsentLibException e){
     private void storeData(){
         storeClient.setConsentUuid(consentUUID);
         storeClient.setMetaData(metaData);
-        storeClient.setTCData(userConsent.TCData);
+        if(userConsent != null) storeClient.setTCData(userConsent.TCData);
```

Comment:
We could initialise `userConsent` with empty values and get rid of this null check. What you think?

Reply:
done